v2/CHANGES

```diff
@@ -145,24 +144,24 @@ Package changes:
 - [ ] intyarpcerrors yarpc/internal/yarpcerrors -> yarpc/v2/internal/internalyarpcerrors
 
 Name changes
-- [ ] {Mock,}{Unary,Stream}Handler -> {Mock,}{Unary,Stream}Transport{Handler,Middleware}
-- [ ] api/transport.* -> v2.*
-- [ ] api/transport/transporttest.* -> v2/yarpctest.*
-- [ ] api/peer.* -> v2.*
-- [ ] api/peer/peertest.* -> v2/yarpctest.*
+- [x] {Mock,}{Unary,Stream}Handler -> {Mock,}{Unary,Stream}Transport{Handler,Middleware}
+- [x] api/transport.* -> v2.*
+- [x] api/transport/transporttest.* -> v2/yarpctest.*
+- [x] api/peer.* -> v2.*
+- [x] api/peer/peertest.* -> v2/yarpctest.*
 - [x] api/middleware.* -> v2/yarpcmiddlware.*
-- [ ] api/middleware/middlewaretest.* -> v2/yarpcmiddlewaretest.*
-- [ ] transport/* -> v2/yarpc*
-- [ ] encoding/* -> v2/yarpc*
-- [ ] yarpc.ErrTansportHasNoReferenceToPeer -> yarpcpeer.ErrDialerHasNoReferenceToPeer
-- [ ] pkg/errors.WrapHandlerError -> v2/yarpcerrors.WrapHandlerError
-- [ ] peer/hostport.PeerIdentifier -> v2.Address
-- [ ] peer/hostport.Peer -> v2/yarpcpeer.AbstractPeer
-- [ ] yarpc.OpentracingTags -> v2/yarpctracing.Tags
-- [ ] internal/errors.ZeroAddrToHostPort -> v2/internal/internalyarpcerrors.ZeroAddrToHostPort
-- [ ] internal/iopool -> v2/internal/internaliopool
-- [ ] TransportAction -> DialerAction
-- [ ] ApplyTransportActions -> ApplyDialerActions
-- [ ] peer.Single -> no longer needed
-- [ ] peer.Binder -> no longer needed
-- [ ] internal/testtime -> v2/internal/internaltesttime
+- [x] api/middleware/middlewaretest.* -> v2/yarpcmiddlewaretest.*
+- [x] transport/* -> v2/yarpc*
+- [x] encoding/* -> v2/yarpc*
+- [x] yarpc.ErrTansportHasNoReferenceToPeer -> yarpcpeer.ErrDialerHasNoReferenceToPeer
+- [x] pkg/errors.WrapHandlerError -> v2/yarpcerrors.WrapHandlerError
+- [x] peer/hostport.PeerIdentifier -> v2.Address
+- [x] peer/hostport.Peer -> v2/yarpcpeer.AbstractPeer
+- [x] yarpc.OpentracingTags -> v2/yarpctracing.Tags
+- [x] internal/errors.ZeroAddrToHostPort -> v2/internal/internalyarpcerrors.ZeroAddrToHostPort
+- [x] internal/iopool -> v2/internal/internaliopool
+- [x] TransportAction -> DialerAction
+- [x] ApplyTransportActions -> ApplyDialerActions
+- [x] peer.Single -> no longer needed
+- [x] peer.Binder -> no longer needed
+- [x] internal/testtime -> v2/internal/internaltesttime
```

Comment:
Let’s just remove the check boxes. These are just notes of things that were done as they were done, originally intended to be copy pasted if you needed a checklist when you brought more code in from v1.